onnx/shape_inference/implementation.h

```diff
@@ -195,6 +195,12 @@ void InferShapes(
     const ISchemaRegistry* schema_registry = OpSchemaRegistry::Instance()
     );
 
+void InferShapes(
+    const std::string& model_path,
+    const bool check_type = false,
```

Comment:
It would be nice to add a comment explaining this flag "check_type". I don't recall and the name itself doesn't clarify it.

Reply:
I was also confused about this... I think it is related to the full_check for checker. I will add some comments for it. Thank you for the suggestion!